pkg/controller/servicemesh/memberroll/controller.go

```diff
@@ -436,9 +440,10 @@ func (r *MemberRollReconciler) reconcileNamespaces(namespacesToReconcile, namesp
 				nsErrors = append(nsErrors, err)
 			}
 		} else {
-			configuredMembers = append(configuredMembers, ns)
+			configured.Insert(ns)
 		}
 	}
+	configuredMembers = configured.List()
```

Comment:
I don't think I understand these changes. The `reconcileNamespaces` function must return a list of namespaces that are configured after the function has done its job, as this list is then written to ˙SMMR.status.configuredMembers˙. With these changes, the function also adds `namespacesToRemove` to the list. 

Oh, did you forget to add `configured.Delete(ns)` after the successful invocation of `removeNamespaceFromMesh()`?

Reply:
Oh. Yes, I had that in a previous version. I squashed commits before pushing and something must have gone wrong